.gitignore

```diff
@@ -1,3 +1,5 @@
 build
 composer.lock
 vendor
+phpcs.xml
+phpunit.xml
```

Comment:
Is it wrong that I have also added phpunit.xml here?

Reply:
@gmponos nope, that's fine 👍